.golangci.yml

```diff
@@ -1,2 +1,15 @@
+linters:
+  enable:
```

Comment:
It's been a little while since I worked with golangci-lint, but IIRC, `enable` replaces rather than adds on to the default set of linters.

in other words, isn't this quietly disabling some useful linters? https://golangci-lint.run/usage/linters/

It's not super well documented at https://golangci-lint.run/usage/configuration/#linters-configuration, but I think I recall seeing that in their issue tracker a few years ago when configuring a new project.

The easy way to test is check the output of `golangci-lint help linters` with this file absent/present.

IIRC, the workaround is to manually specify the default linters as well.

Reply:
Awesome thanks for clarifying!